config-sample.yaml

```diff
@@ -118,7 +118,7 @@ keys:
 #   max_participants: 0
 #   # only accept specific codecs for clients publishing to this room
 #   # this is useful to standardize codecs across clients
-#   # other supported codecs are video/h264, video/vp9
```

Comment:
This and the change below is from a different PR. Sorry, it is here because I branched off that PR's branch.